library/get_config.py

```diff
@@ -31,6 +31,12 @@
     choices:
       - running
       - startup
+  flags:
+    description:
+      - Identifies the particular section of configuration that should be fetched from
+        remote host
+    type: list
+    aliases: ['filter']
   format:
```

Comment:
@ganeshrn Can you please add example for flags before we merge this?

Reply:
fixed